--- a/apicaro/__init__.py
+++ b/apicaro/__init__.py
@@ -24,9 +24,9 @@
     y recibir datos.
     una ves inicializada la clase, hay que tener en cuenta lo siguiente:
     la variable PUERTO, es donde se carga el valor del dispositivo serie.
-    por defecto es /dev/ttyACM0. cambiarla antes de usar la funcion iniciar()
+    por defecto es /dev/icaro. cambiarla antes de usar la funcion iniciar()
     """
-    PUERTO='/dev/ttyACM0' # valor inicial por defecto
+    PUERTO='/dev/icaro' # valor inicial por defecto
     BAUDIOS=9600
     BYTESIZE=8
     PARITY='N'
@@ -43,7 +43,7 @@
         """
         abre el puerto, setea todos los valores por defecto.
         sus variables por defecto son:
-        PUERTO='/dev/ttyUSB0' # valor inicial por defecto
+        PUERTO='/dev/icaro' # valor inicial por defecto
         BAUDIOS=9600
         BYTESIZE=8
         PARITY='N'
--- a/apicaro/apicaro.py
+++ b/apicaro/apicaro.py
@@ -27,9 +27,9 @@
     y recibir datos.
     una ves inicializada la clase, hay que tener en cuenta lo siguiente:
     la variable PUERTO, es donde se carga el valor del dispositivo serie.
-    por defecto es /dev/ttyACM0. cambiarla antes de usar la funcion iniciar()
+    por defecto es /dev/icaro. cambiarla antes de usar la funcion iniciar()
     """
-    PUERTO='/dev/ttyACM0' # valor inicial por defecto
+    PUERTO='/dev/icaro' # valor inicial por defecto
     BAUDIOS=9600
     BYTESIZE=8
     PARITY='N'
@@ -46,7 +46,7 @@
         """
         abre el puerto, setea todos los valores por defecto.
         sus variables por defecto son:
-        PUERTO='/dev/ttyUSB0' # valor inicial por defecto
+        PUERTO='/dev/icaro' # valor inicial por defecto
         BAUDIOS=9600
         BYTESIZE=8
         PARITY='N'
--- a/plugintortucaro/icaro/icaro.py
+++ b/plugintortucaro/icaro/icaro.py
@@ -22,7 +22,7 @@
 import apicaro
 import time
 puerto = apicaro.puerto()
-puerto.PUERTO='/dev/ttyUSB0'
+puerto.PUERTO='/dev/icaro'
 puerto.iniciar()
 class Icaro(Plugin):
     def __init__(self, parent):
@@ -56,7 +56,7 @@
                      style='basic-style-1arg',  
                      label=_('abrir puerto'),  
                      prim_name='abrir',
-                     default=['/dev/ttyUSB0'],  
+                     default=['/dev/icaro'],  
                      help_string=_('abre el puerto para comunicacion con la placa'))
         self._parent.lc.def_prim('abrir', 1, lambda self, valor: primitive_dictionary['abrir'](valor))
 
--- a/udev/26-microchip.rules
+++ b/udev/26-microchip.rules
@@ -1,5 +1,5 @@
 #PICKit 1
-ATTR{idVendor}=="04d8", ATTR{idProduct}=="feaa", MODE="0660",GROUP="microchip"
+ATTR{idVendor}=="04d8", ATTR{idProduct}=="feab", MODE="0660", GROUP="microchip", SYMLINK+="icaro"
 
 
 
--- a/udev/41-microchip.rules
+++ b/udev/41-microchip.rules
@@ -1,5 +1,5 @@
 # Pinguino
-ATTR{idVendor}=="04d8", ATTR{idProduct}=="feaa", MODE="0660",GROUP="microchip"
+ATTR{idVendor}=="04d8", ATTR{idProduct}=="feaa", MODE="0660", GROUP="microchip", SYMLINK+="icaro"
 
 
 
